test/validator/astnode-cp/ASTNodeOperation.test.ts

```diff
@@ -272,12 +272,12 @@ describe('ASTNodeOperation', () => {
 					['!4.2e+1;',              VALUE.FALSE],
 					['!"";',                  VALUE.FALSE],
 					['!"hello";',             VALUE.FALSE],
-					['![];',                  VALUE.FALSE],
-					['![42];',                VALUE.FALSE],
-					['![a= 42];',             VALUE.FALSE],
-					['!List.<int>([]);',      VALUE.FALSE],
-					['!List.<int>([42]);',    VALUE.FALSE],
-					['!Dict.<int>([a= 42]);', VALUE.FALSE],
+					['!();',                  VALUE.FALSE],
+					['!(42,);',               VALUE.FALSE],
+					['!(a= 42);',             VALUE.FALSE],
+					['!List.<int>(());',      VALUE.FALSE],
+					['!List.<int>((42,));',   VALUE.FALSE],
+					['!Dict.<int>((a= 42));', VALUE.FALSE],
```

Comment:
```suggestion
					['![];',                  VALUE.FALSE],
					['![42];',                VALUE.FALSE],
					['![a= 42];',             VALUE.FALSE],
```
same for `?` operator below